src/loader/loader.c

```diff
@@ -470,6 +470,41 @@ post_analyze_hook(ParseState *pstate, Query *query, JumbleState *jstate)
 	}
 }
 
+/*
+ * Check if a string is an UUID and error out otherwise.
+ */
+static void
+check_uuid(const char *label)
+{
+	const MemoryContext oldcontext = CurrentMemoryContext;
+	const char *uuid = strchr(label, SECLABEL_DIST_TAG_SEPARATOR);
+	if (!uuid || strncmp(label, SECLABEL_DIST_TAG, uuid - label) != 0)
+		ereport(ERROR,
+				(errmsg("TimescaleDB label is for internal use only"),
+				 errdetail("Security label is \"%s\".", label),
+				 errhint("Security label has to be of format \"dist_uuid:<UUID>\".")));
```

Comment:
My previous comment about dropping the hint still applies.

Reply:
I prefer to have this present for debugging purposes and trust users rather than to hide useful information that could help in resolving an issue.